### DISCUSSION TITLE
bug: Not visible lines on kitty terminal sometimes.

### DISCUSSION THREAD

**Discussion by Tokariew** [eza-community/eza#178] [Q&A, Howto, Help] [answered]
On kitty terminal if background and color0 are the same, then the lines in size column or in permission colours are not visible, is it possible to have option to change them or set. On exa it worked fine

Ubuntu 20.04, kitty 0.28.1 created by Kovid Goyal, eza from nix-env 

eza - A modern, maintained replacement for ls
v0.10.8 [+git]
https://github.com/eza-community/eza

---
```
 ➜ eza --long
.rw-rw-r--  10k dkorbuszewski 18 Jul 15:34 conf22.txt
drwxr-xr-x    - dkorbuszewski 11 Aug 12:04 Desktop
drwxr-xr-x    - dkorbuszewski 28 Aug 10:57 Documents
drwxr-xr-x    - dkorbuszewski 24 Aug 13:06 Downloads
drwxrwxr-x    - dkorbuszewski 28 Aug 11:58 gits
drwxrwxr-x    - dkorbuszewski 17 Aug 08:59 go
drwxrwxrwx    - root          28 Aug 12:28 labnet
.rw-rw-r--  38k dkorbuszewski 24 Aug 15:51 log-error.log
.rw-rw-r-- 207k dkorbuszewski 24 Aug 16:20 log.txt
drwxrwxr-x    - dkorbuszewski 31 Jul 16:44 logs
.rw-rw-r-- 2.0k dkorbuszewski 28 Aug 12:51 minicom.log
drwxr-xr-x    - dkorbuszewski  8 May 10:06 Music
drwxr-xr-x    - dkorbuszewski 23 Jun 13:58 Pictures
drwxr-xr-x    - dkorbuszewski  8 May 10:06 Public
drwxr-xr-x    - dkorbuszewski  2 Aug 15:56 Templates
drwxrwxr-x    - dkorbuszewski 18 May 11:54 ti
drwxrwxr-x    - dkorbuszewski 17 Jul 09:06 tmp
drwxr-xr-x    - dkorbuszewski  2 Aug 15:56 Videos

```

![image](https://github.com/eza-community/eza/assets/7306237/d06409be-118e-4119-bca7-3822d9d16eea)

**Comment by cafkafk** [comment, marked as answer]
The  `EXA_COLORS`  environment variable can be used to customise the colours that eza uses to highlight file names, file metadata, and parts of the UI.

(Yes, it's still named EXA for backwards compatibility, but might change soon)

You should check out the manual page via e.g. `man eza_colors`.

If this doesn't work for some reason do tell me!

**Reply by Tokariew** [reply]
Thanks, after some reading fixed lines for my config. Needed to change `xx`

**Reply by planet36** [reply]
What value of `EXA_COLORS` restores the old behavior?
I've looked at `eza_colors` and experimented with values such as `EXA_COLORS='gr=32:gw=32:gx=32'`, but I wasn't successful in changing anything.

**Reply by smackysnacks** [reply]
@planet36 - `EXA_COLORS="xx=''"`.
